### PR TITLE
0.13 cherry picks from `main`

### PR DESCRIPTION
**Pull request by vvagaytsev**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: plugins/terraform/cli.ts [hunk 1]
```diff
@@ -209,12 +208,13 @@ export const terraformCliSpecs: { [version: string]: PluginToolSpec } = {
     name: "terraform-1-2-9",
     description: "The terraform CLI, v1.2.9",
     type: "binary",
+    _includeInGardenImage: true,
```
Comment (edvald) [inline plugins/terraform/cli.ts:211]: Why was this moved?

Comment (vvagaytsev) [reply plugins/terraform/cli.ts:211]: I'm not sure, it was moved in the [original commit](https://github.com/garden-io/garden/commit/d5441475dccfc37701d096b44c994112b7f2f228). I assume that we apply `_includeInGardenImage: true` only to the latest available tool version, don't we?

Comment (vvagaytsev) [reply plugins/terraform/cli.ts:211]: cc @thsig

**Review by edvald — APPROVED** [review]
